docs/metadata/git/git-commit.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Commit changes
+id: git-commit
+description: Source Control
+sidebar_position: 2
+tags:
+  - metadata
+  - Git
+  - commit
+---
+
+This page describes how to commit your changes and create new branches from the main branch.
+
+## Commit changes into the dev branch
+
+When you create your first Project, `dev` branch is automatically created for you. This is to ensure that no edits/commits are made directly on the `main` [branch](/concepts/Project/#Project-commits).
```

Comment:
This is incorrect. We do not create any branches automatically for the user

Reply:
We only auto-suggest dev/$username for new branch creation if it is a valid branch

Reply:
Done.